features/support/cucumber_rails_helper.rb

```diff
@@ -4,7 +4,7 @@
 require 'cucumber'
 require 'capybara'
 
-module CucumberRailsHelper
+module CucumberRailsHelper # rubocop:todo Layout/ModuleLength
```

Comment:
Wherever possible we're trying to avoid doing in-line exclusions and using the top level control with a comment explaining why.
 
 Ideally longer term we should compress this down a bit. But it is test only code, so it's low prio.

Reply:
I fixed this by removing a superfluous line in [`7f667a8`](https://github.com/cucumber/cucumber-rails/pull/559/commits/7f667a82df3a01346c0aaf2efbab0b4630f5414a).